src/sentry/tasks/reports.py

```diff
@@ -160,30 +161,30 @@ def merge_series(target, other, function=operator.add):
     return results
 
 
+def __get_range_chunked(issue_ids, start, stop, rollup):
```

Comment:
Is there a reason not to generalize this and `__get_sums_chunked`? The implementations are basically identical, aside from the function called and the chunk size.